dotsloader/src/main/java/com/agrawalsuneet/dotsloader/basicviews/CircularLoaderBaseView.kt

```diff
@@ -31,41 +34,40 @@ open class CircularLoaderBaseView : DotsLoaderBaseView {
         this.bigCircleRadius = bigCircleRadius
         this.defaultColor = dotsColor
 
-        initCordinates()
+        initCoordinates()
         initPaints()
     }
 
     constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {
         initAttributes(attrs)
-        initCordinates()
+        initCoordinates()
         initPaints()
         initShadowPaints()
     }
 
     constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {
         initAttributes(attrs)
-        initCordinates()
+        initCoordinates()
         initPaints()
         initShadowPaints()
     }
 
-    override fun initAttributes(attrs: AttributeSet) {
+    final override fun initAttributes(attrs: AttributeSet) {
```

Comment:
try to avoid making this function final as if someone wants to extend this class and override its functionality, he or she can be able to do it.